openlibrary/templates/admin/sponsorship.html

```diff
@@ -0,0 +1,137 @@
+$def with (summary)
+
+$ _x = ctx.setdefault('bodyid', 'admin')
+$ _x = ctx.setdefault('usergroup', 'admin')
+
+$ statuses = {0: 'Needs purchasing', 1: 'Needs digitizing', 2: 'Needs republishing', 3: 'Complete'}
+
+<div id="contentHead">
+  $:render_template("admin/menu")
+  <h1>Book Sponsorship</h1>
+</div>
+
+<div id="contentBody">
```

Comment:
Give this a unique class name; `page-sponsorship` maybe? That way we can scope all the related CSS to this specific table, and avoid it leaking out.